go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.2
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/hashicorp/golang-lru v0.5.4
+	github.com/lazyledger/lazyledger-core v0.0.0-20210115223437-eff282ad2592
```

Comment:
One general remark on the module (the cosmos-sdk one): I was wondering if we should rename it from
`module github.com/cosmos/cosmos-sdk` to `module github.com/lazyledger/cosmos-sdk`. 
I'm almost surprised to see that the replace directive worked in the ll-app without a problem in this case. 

Let's update the app when this is merged and see if this works without any further changes.